walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,93 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Arrays;
 
 public final class FindMeetingQuery {
+  /*
+  * Goal: Return a collection of free time ranges
+  * Free time ranges: time ranges where no meeting attendees have an event scheduled,
+  * and it is long enough to conduct the whole meeting.
+  *
+  **/
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
Your code assumes that events are ordered. I think it would probably be best to sort within your function (or at a minimum document the expectation).

(if I missed something in project spec you can ignore)